tools/scxtop/benches/search_benchmark.rs

```diff
@@ -0,0 +1,93 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use scxtop::available_perf_events;
+use scxtop::Search;
+
+fn get_search() -> Search {
+    let available_perf_events_list: Vec<String> = available_perf_events()
+        .unwrap()
+        .iter()
+        .flat_map(|(subsystem, events)| {
+            events
+                .iter()
+                .map(|event| format!("{}:{}", subsystem.clone(), event.clone()))
+        })
+        .collect();
```

Comment:
nit: you don't need the clones here
```suggestion
    let available_perf_events_list: Vec<String> = available_perf_events()
        .unwrap()
        .into_iter()
        .flat_map(|(subsystem, events)| {
            events
                .into_iter()
                .map(move |event| format!("{}:{}", &subsystem, &event))
        })
        .collect();
```